package.json

```
@@ -19,7 +19,7 @@
   "peerDependencies": {
     "react-native": "*",
     "react-native-sensors": "*",
-    "react-native-webview": "*",
+    "@metamask/react-native-webview": "*",
```

```suggestion
    "@metamask/react-native-webview": ">=14.0.1",
```

Since we know `14.0.0` to be broken.